docs/pages/Introduction/bridges.mdx

```diff
@@ -0,0 +1,69 @@
+{ /* explain the interoprability behind bridges */}
+
+
+## Traditional way of accessing other networks
+
+Bridges provide a way to access other networks without having to go through a centralized exchange. This is important because centralized exchanges requires KYC that could take days depending on the amount of tokens you want to transfer, they are expensive and could be unreliable sometimes. They can also be hacked or shut down by regulators.
+
+
+## Benefits of bridges
+Bridges facilitate a multitude of applications by enabling blockchain networks to share data and transfer assets among themselves. They foster the growth of the broader crypto ecosystem by empowering blockchains to harness each other's innovations.
+
+Developers benefit from bridges in the following ways:
+
+- Enabling the transfer of data, information, and assets across multiple chains.
+- Expanding the design space for protocols, thus introducing new features and use cases. For instance, a protocol initially deployed on Ethereum Mainnet for yield farming could extend its offerings to liquidity pools across all EVM-compatible chains.
+- Leveraging the unique strengths of different blockchains. For instance, developers can take advantage of lower fees offered by various Layer 2 solutions by deploying their decentralized applications (dApps) across rollups and sidechains, with users bridging across them.
+- Fostering collaboration among developers from diverse blockchain ecosystems to create innovative products.
+- Drawing users and communities from various ecosystems to their dApps. For instance, a DeFi protocol deployed on Ethereum could attract users from Binance Smart Chain by offering cross-chain asset transfers through a bridge.
+
+{ /* write five benefits of bridges */}
+
+
+
+## Types of bridges
+At a high level, bridges can be categorized as trusted and trustless.
+
+{   /* Broad category */}
+
+- **Oracle based bridges(Trusted)**: These are infrastructure solutions that enable interoperability and communication between disparate blockchain networks by leveraging external data sources known as oracles. These bridges facilitate the exchange of information and assets between blockchains that may have different protocols and consensus mechanisms. They rely on trusted set of accounts or entities to authorise and verify transactions and state transitions across interconnected networks.
+
+
+- **Consensus based bridges(Trustless)**: These are infrastructure solutions that enables interoperability between different blockchain networks through a mechanism based on achieving consensus among the participating networks. Unlike oracle-based bridges, which rely on external data sources to facilitate communication between blockchains, consensus-based bridges use consensus algorithms to ensure the validity and integrity of transactions and state transitions across interconnected networks. This is the most secure way to connect two networks.
```

Comment:
This should be reworded to better communicate the technical details of consensus bridges, succinctly.
Check the polytope lab articles for references.